src/sources/geniuscom.cpp

```diff
@@ -1,14 +1,16 @@
 #include "stdafx.h"
 #include <cctype>
 
-#include "pugixml.hpp"
+#include "cJSON.h"
 
 #include "logging.h"
 #include "lyric_source.h"
 #include "tag_util.h"
 
 static const GUID src_guid = { 0xb4cf497f, 0xd2c, 0x45ff, { 0xaa, 0x46, 0xf1, 0x45, 0xa7, 0xf, 0x90, 0x14 } };
 
+constexpr int resultLimit = 3;

```

Comment:
Minor: constants use UPPER_SNAKE_CASE

```suggestion
constexpr int RESULT_LIMIT = 3;
```